test/integration/targets/pid/tasks/main.yml

```diff
@@ -0,0 +1,17 @@
+# Test code for the pid module
+# Copyright 2018, Saranya sridharan
+
+
+- name: "Checking empty result" 
+  pid: 
+    name: "blahblah"
+  register: emptypids
+     
+- name: "Verify the process id returned is empty"
+  assert:
+    that:
+    - emptypids is not changed
+    - emptypids.pid == [""] 
```

Comment:
If the process isn't running, the result should not be a list containing an empty string, but an empty list!